src/wp-includes/user.php

```diff
@@ -1826,6 +1826,34 @@ function sanitize_user_field( $field, $value, $user_id, $context ) {
 	return $value;
 }
 
+/**
+ * Adds any users from the given IDs to the cache that do not already exist in cache.
+ *
+ * @since 6.1.0
+ * @access private
+ *
+ * @see update_user_caches()
+ *
+ * @global wpdb $wpdb WordPress database abstraction object.
+ *
+ * @param array $ids               ID list.
+ * @param bool  $update_meta_cache Optional. Whether to update the meta cache. Default true.
+ */
+function _prime_user_caches( $ids, $update_meta_cache = true ) {
```

Comment:
Note to reviewer. This is different from `cache_users`. User meta is forced to be primed and is not plugable.